{"uri":"file:///Users/vega/Code/huli/manager-web/src/vendor/aws/aws-sdk-php/src/Aws/S3/Exception/DeleteMultipleObjectsException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Aws\\S3\\Exception","location":[16,0,16,27]},{"kind":1,"name":"Aws\\S3\\Exception\\DeleteMultipleObjectsException","location":[21,0,47,1],"associated":[{"kind":1,"name":"Aws\\S3\\Exception\\S3Exception"}],"description":"Exception thrown when errors occur in a DeleteMultipleObjects request","children":[{"kind":16,"name":"$errors","modifiers":2,"location":[26,14,26,31],"description":"Array of errors","scope":"Aws\\S3\\Exception\\DeleteMultipleObjectsException","type":"array"},{"kind":32,"name":"__construct","modifiers":1,"location":[31,4,35,5],"scope":"Aws\\S3\\Exception\\DeleteMultipleObjectsException","children":[{"kind":128,"name":"$errors","location":[31,32,31,55],"description":"Array of errors","scope":"__construct","value":"array()","type":"array"}]},{"kind":32,"name":"getErrors","modifiers":1,"location":[43,4,46,5],"description":"Get the errored objects","scope":"Aws\\S3\\Exception\\DeleteMultipleObjectsException","type":"array"}]}]}}